ui/appui-react/src/appui-react/content/ContentLayout.tsx

```diff
@@ -15,14 +15,14 @@ import { SplitPane } from "./split-pane/SplitPane";
 import type { CommonProps } from "@itwin/core-react";
 import { Orientation } from "@itwin/core-react";
 import type { ContentGroup } from "./ContentGroup";
-import type {
-  ContentLayoutProps,
-  LayoutFragmentProps,
-  LayoutHorizontalSplitProps,
-  LayoutSplitPropsBase,
-  LayoutVerticalSplitProps,
+import {
+  type ContentLayoutProps,
+  type LayoutFragmentProps,
+  type LayoutHorizontalSplitProps,
+  type LayoutSplitPropsBase,
+  type LayoutVerticalSplitProps,
+  UiEvent,
 } from "@itwin/appui-abstract";
-import { UiEvent } from "@itwin/appui-abstract";
 import type { ActiveContentChangedEventArgs } from "../framework/FrameworkContent";
```

Comment:
Although I find import reordering annyoing to review, I understand it, but here we are changing the way we import stuff, but only for one package, is there a reason for that ? Should prettier catch that ? I feel there is a lot of reorder in this PR in files that are not otherwise modified, that makes it a bit lenghty to review.

Reply:
I agree. My prettier settings were automatically "correcting" to mixing type and value imports, and then I was undisciplined and sometimes followed the new syntax. I'll revert changes to files not otherwise modified which I most likely saved while reading.

Reply:
Done